src/components/vuestic-components/va-avatar/VaAvatar.vue

```diff
@@ -1,26 +1,74 @@
 <template>
-  <div class="va-avatar">
-    <slot/>
+  <div class="va-avatar" :style="computedStyle">
+    <va-icon v-if="icon" :name="icon" />
+    <img v-if="src" :src="src" />
+    <span v-if="!icon && !src"><slot/></span>
   </div>
 </template>
 
 <script>
+import { SizeMixin } from '../../../mixins/SizeMixin'
+import { ColorThemeMixin } from '../../../services/ColorThemePlugin'
+import VaIcon from '../va-icon/VaIcon'
+
 export default {
   name: 'va-avatar',
+  mixins: [SizeMixin, ColorThemeMixin],
+  components: {
+    VaIcon,
+  },
+  props: {
+    color: {
```

Comment:
I think this one is added by the color mixin

Reply:
fixed